### PR TITLE
Rename webhooks to hooks

### PR DESCRIPTION
**Pull request by adamruzicka**
Opening as a draft since the changes need to be coordinated across projects to minimize downtime

### CODE REVIEW

File: src/js/nav/globalNav.js [hunk 1]
```diff
@@ -25,9 +25,9 @@ export const grouppedNav = {
                 title: 'Settings',
                 subItems: [
                     {
-                        id: 'webhooks',
-                        title: 'Webhooks',
-                        reload: 'webhooks'
+                        id: 'hooks',
+                        title: 'Hooks',
+                        reload: 'hooks'
```
Comment (kruai) [inline src/js/nav/globalNav.js]: To follow the pattern for Settings apps, this should probably be `reload: 'settings/hooks'`.

**Comment by iphands** [comment]
@adamruzicka we are going to change the appid too... your assets will be at /apps/hooks and /beta/apps/hooks so a small change on your app is required.

**Comment by adamruzicka** [comment]
I was counting on that, we have a PR ready to handle it when it happens

**Comment by kruai** [comment]
@adamruzicka in addition to the comment I made above, the `config/index_map.json` file should also be updated accordingly.

**Comment by adamruzicka** [comment]
It should be good to go from our side, things work differently under the hood than I thought they would.

**Pull request merged**